README.md

```diff
@@ -2,31 +2,142 @@
+We used the `execute` method to execute `CREATE TABLE` and `DROP TABLE` statements,
+and the `executeUpdate` method to execute `INSERT`, `UPDATE`, and `DELETE` statements.
 
-## Topic 1
+In this lesson we will use the `executeQuery` method to execute a `SELECT` statement.
+The `executeQuery` method executes the query passed as a parameter, and returns a single `ResultSet` object:
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+- `ResultSet	executeQuery(String sql)`
 
-## Topic 2
+The `ResultSet` interface encapsulates a data table:
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+- **ResultSet** : An interface that represents a data table, which is usually generated by executing a `SELECT` statement.
+
+A `ResultSet` object maintains a cursor pointing to its current row of data.
+The cursor is initially positioned before the first row.  We use the `next()`
+method to move the cursor forward to the next row. The `next()` method returns a boolean
+indicating whether there are more rows in the result set. We call `next()` in a loop
+to move the cursor forward to each row in the query result set.
+
+- `boolean	next()` : Moves the cursor forward one row from its current position.
+
+The `ResultSet` interface includes methods to retrieve a value from the current row either by column index or name.
+We will use the following methods:
+
+- `int	getInt(int columnIndex)` : Returns an integer value based on the column index.
+- `int	getInt(int columnName)` : Returns an integer value based on the column name.
+- `double	getDouble(int columnIndex)` : Returns a double value based on the column index.
+- `double	getDouble(int columnName)` : Returns a double value based on the column name.
+- `String	getString(int columnIndex)` : Returns a String value based on the column index.
+- `String	getString(int columnName)` : Returns a String value based on the column name.
+
+## Code along
+
+[Fork and clone](https://github.com/learn-co-curriculum/java-mod-5-jdbc-resultset) this lesson to run the sample Java programs.
+
+NOTE: This lesson assumes you created a new PostgreSQL database named `employee_db` as instructed
+in the first lesson.
+
+The lesson files include the `CreateTableStatement` and `InsertStatement` classes
+from the previous lesson.  Run the `main` method in each class to ensure the `employee` table
+is populated with the three sample rows.
+
+1. Run `CreateTableStatement.main` to create an empty `employee` table.
+2. Run `InsertStatement.main` to add three rows to the table.
+
+Query the `employee` table in the **pgAdmin** query tool to confirm the 3 rows:
+
+![insert 3 rows](https://curriculum-content.s3.amazonaws.com/6036/jdbc-statement/insert_rows.png)
+
+## Execute an SELECT statement
```

Comment:
Grammar error: should be "Execute a SELECT statement" instead of "an"

Reply:
thanks, fixed.